sklearn/covariance/_elliptic_envelope.py

```diff
@@ -171,6 +178,7 @@ def fit(self, X, y=None):
         self : object
             Returns the instance itself.
         """
+        self._validate_params()
         if self.contamination != "auto":
```

Comment:
Remove all this `if` block. It will be handle by `_validate_params`

Reply:
done